aggregating-profiler/src/main/java/ru/fix/aggregating/profiler/Tagger.java

```diff
@@ -0,0 +1,12 @@
+package ru.fix.aggregating.profiler;
+
+/**
+ *
+ * @author Andrey Kiselev
+ */
+
+public interface Tagger {
+    static final String DEFAULT_TAG_VALUE = "defval";
+    <T extends Tagged> T assignTag(String profiledCallName, T tagged);
```

Comment:
default tag name inside Tagged interface is confusing.   What case it used for? - no clear from first glance.  
Tagger interface is API for tag assignment.  Maybe it would be better not to burden Tagger interface with knowledge that there is a case where we use RegexpTagger that set this DEFAULT tag on Tagged object?

Suggestion: 
1.
remove DEFAULT_TAG_KEY from  Tagged
remove DEFAULT_TAG_VALUE from Tagger

2.
RegexpTagger is responsible for setting tags based on it's configuration. 
We can require that both tag name and tag value is provided by RegexpTagger: 
public static final String RegexpTagger.RATE_TAG

3. 
buildReportAndReset do not know of particular tagger implementation. Instead it simply asks user to specify which tag name and value to use in filter:
buildReportAndReset(String tagKey, String tagValue)

This way User decided to use RegexpTagger
profiler.setTagger(new RegexpTagger)
User knows, that now metrics are tagged by RegexpTagger
so User will filter them during report build phase: 
reporter.buildReportAndReset(RegexpTagger.RATE_TAG, rateTagValue)